src/Sylius/Bundle/ApiBundle/CommandHandler/Account/ChangeShopUserPasswordHandler.php

```diff
@@ -17,10 +17,11 @@
 use Sylius\Component\Core\Model\ShopUserInterface;
 use Sylius\Component\User\Repository\UserRepositoryInterface;
 use Sylius\Component\User\Security\PasswordUpdaterInterface;
-use Symfony\Component\Messenger\Handler\MessageHandlerInterface;
+use Symfony\Component\Messenger\Attribute\AsMessageHandler;
 use Webmozart\Assert\Assert;
 
-final readonly class ChangeShopUserPasswordHandler implements MessageHandlerInterface
+#[AsMessageHandler]
+final readonly class ChangeShopUserPasswordHandler
```

Comment:
It's unnecessary, as it has no effect. It's used by autoconfiguration which is disabled for bundles.

Reply:
Yeah indeed, but Symfony uses it on Symfony\Component\Cache\Messenger\EarlyExpirationHandler.
Maybe it's just an information.

BTW, on AsCommand attribute, it's now required, maybe the'll force using it at a moment.